go/signedexchange/README.md

```diff
@@ -25,7 +25,7 @@ go get -u github.com/WICG/webpackage/go/signedexchange/cmd/...
 ### Creating our first signed exchange
 In this section, we guide you to create a signed exchange file that is signed using a self-signed certificate pair.
 
-Here, we assume that you have an access to an HTTPS server capable of serving static content. [1] Please substitute `https://yourcdn.example.net/` URLs to your web server URL.
+Here, we assume that you have an access to an HTTPS server capable of serving static content. [1] Please substitute `https://yourcdn.example.net/` URLs to your web server URL, and `example.org` to the domain you own to sign exchange for.
```

Comment:
Maybe "to the domain you own and want to sign the exchange for" or "to the domain for which you want to sign the exchange". I'm not sure how much we want to emphasize domain ownership vs just assuming that people can only get certs for their own domains.

Reply:
Thanks. Suggestion taken.